File: C-Chat-API/C-Chat-API/Models/Entities/Message.cs
namespace C_Chat_API.Models.Entities
{
    public class Message
    {
        public int MessageId { get; set; }
        public DateTime Date { get; set; }
        public string Content { get; set; }

        // Foreign keys
        public User User { get; set; }
        public Chat Chat { get; set; }
    }
}

File: C-Chat-Web/src/app/model/classes/message.ts
export class Message {
  author: string = '';
  chatName: string = '';
  date: string = '';
  content: string = '';

  constructor(author: string, chatName: string, date: string, content: string){
    this.author = author;
    this.chatName = chatName;
    this.date = date;
    this.content = content;
  }
}
